src/main/kotlin/net/atos/zac/app/admin/model/RestReferenceTable.kt

```diff
@@ -4,6 +4,7 @@
  */
 package net.atos.zac.app.admin.model
 
+import jakarta.validation.Valid
 import jakarta.validation.constraints.NotBlank
 import net.atos.zac.admin.model.ReferenceTable
```

Comment:
Unused?

Reply:
That would be weird because Detekt runs fine.. Ah,it is used.